### ISSUE TITLE
Multiple Data Sets

### ISSUE THREAD

**Issue by floydba** [s-u/RJDBC#51]
Does this version of RJDBC allow for return of multiple datasets. I am trying to run a stored procedure on a database and I am getting the following  error:

java.sql.PreparedStatement.executeQuery() cannot be called because multiple result sets were returned

**Comment by s-u** [comment]
No, the current version only supports a single result set since it uses `executeQuery()`. However, it is not unreasonable to add that functionality.
